easybuild/easyblocks/s/suitesparse.py

```diff
@@ -172,8 +181,11 @@ def configure_step(self):
 
         else:
             # after v6.0.0, no option for metis, its own metis is used anyway
-            # nothing to do here, set the CMAKE_OPTIONS in easyconfigs
-            pass
+            # set CMAKE_OPTIONS if it is not specified in easyconfigs
```

Comment:
Should probably verify that CMake is a builddep too, just to make sure it will actually use the EB installed CMake and not a system cmake.